Automattic/wp-calypso
# Change source of Google Workspace `Alternate email` field in checkout

#### Proposed Changes

When buying Google Workspace for an existing domain, we ask users to provide an `Alternate email address` on the checkout page. This detail is needed to set up a Google Workspace customer. The problem is that this field is never pre-populated and users always have to manually enter their details there.

With this PR, we stop relying on the `alternate-email` field that was previously added specifically when users bought Google Workspace for an existing domain. Instead, we use the `email` field, which is what we already display for users who are buying a new domain+Google Workspace. This ensures that the field is correctly pre-populated and that users aren't confronted with an error message as the first thing they see on the checkout page.

This PR is complemented by a back-end patch (D88790-code), **but it does not depend on that change**.

For some additional background, see pbLl1t-1c9-p2. This is also tangentially related to #67420.

| Before | After |
| - | - |
| ![gws-alternate-email-before](https://user-images.githubusercontent.com/1101677/192961858-775a3b39-8d0d-48d8-a47c-f30ec69fb6a5.png) | ![gws-alternate-email-after](https://user-images.githubusercontent.com/1101677/192961877-fa237caf-1f0d-44c0-934f-4658bc189089.png) |

#### Testing Instructions



1. Have an existing domain without an email solution.
2. Proceed to purchase a Google Workspace subscription for that domain.
3. On the checkout page, ensure that you are not immediately confronted with an error message about validation.
4. Also ensure that you see an email address under the `Google Workspace account information` heading. Remember that email address.
5. Purchase the subscription and wait for account setup to complete. 
6. Log in to Google and verify that the alternate email address matches what you saw during checkout. You can do this by looking at `Secondary email` on https://admin.google.com/u/0/ac/accountsettings/profile

#### Pre-merge Checklist






Related to #
